__tests__/assignee.test.js

```diff
@@ -0,0 +1,114 @@
+const Helper = require('../__fixtures__/helper')
+const assignee = require('../lib/assignee')
+const Configuration = require('../lib/configuration')
+
+test('that mergeable is false when less than minimum', async () => {
+  let validation = await assignee(createMockPR(1), null, config({min: 2}))
+  expect(validation.mergeable).toBe(false)
+})
+
+test('that mergeable is true when the same as minimum', async () => {
+  let validation = await assignee(createMockPR(2), null, config({min: 2}))
+  expect(validation.mergeable).toBe(true)
+})
+
+test('that mergeable is true when greater than minimum', async () => {
+  let validation = await assignee(createMockPR(3), null, config({min: 2}))
+  expect(validation.mergeable).toBe(true)
+})
+
+test('that description is dynamic based on minimum', async () => {
+  let validation = await assignee(createMockPR(1), null, config({min: 5}))
+  expect(validation.description[0]).toBe('Assignee count is less than "5"')
+})
+
+test('that description is null when mergeable', async () => {
+  let validation = await assignee(createMockPR(5), null, config({min: 5}))
+  expect(validation.description).toBe(null)
+})
+
+test('checks that advance_setting min is working', async () => {
+  let configuration = `
+  mergeable:
+    assignee:
+      min: 2
+  `
+
+  let validation = await assignee(createMockPR(1), null, config({config: configuration}))
+  expect(validation.mergeable).toBe(false)
+  expect(validation.description[0]).toBe('Assignee count is less than "2"')
+
+  validation = await assignee(createMockPR(2), null, config({config: configuration}))
+  expect(validation.mergeable).toBe(true)
+})
+
+test('checks that advance_setting max is working', async () => {
+  let configuration = `
+  mergeable:
+    assignee:
+      min: 2
+  `
+
+  let validation = await assignee(createMockPR(1), null, config({config: configuration}))
+  expect(validation.mergeable).toBe(false)
+  expect(validation.description[0]).toBe('Assignee count is less than "2"')
```

Comment:
should be using max description.

Reply:
removed the whole test case as it is a duplicate